scripts/vim-patch.sh

```diff
@@ -124,13 +124,22 @@ commit_message() {
   fi
 }
 
-find_git_remote() {
-  git_remote=$(git remote -v \
+find_nvim_remote() {
+  nvim_remote=$(git remote -v \
     | awk '$2 ~ /github.com[:\/]neovim\/neovim/ && $3 == "(fetch)" {print $1; exit}')
-  if [[ -z "$git_remote" ]]; then
-    git_remote="origin"
+  if [[ -z "$nvim_remote" ]]; then
+    nvim_remote="origin"
   fi
-  echo "$git_remote"
+  echo "$nvim_remote"
+}
+
+find_fork_remote() {
```

Comment:
these functions are almost identical, can be easily parameterized. something like

```
local r
if [ "$1" = fork ] ; then
  r=$(git remote -v | awk '$2 !~ /github.com[:\/]neovim\/neovim/ && $3 == "(fetch)" {print $1; exit}')
else
  r=$(git remote -v | awk '$2 ~ /github.com[:\/]neovim\/neovim/ && $3 == "(fetch)" {print $1; exit}')
fi
```